pkg/network/protocols/http/telemetry.go

```diff
@@ -11,15 +11,16 @@ package http
 import (
 	"time"
 
-	"go.uber.org/atomic"
-
 	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
+	nettelemetry "github.com/DataDog/datadog-agent/pkg/network/telemetry"
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
-type telemetry struct {
-	then *atomic.Int64
+var (
+	then = nettelemetry.NewStatGaugeWrapper("usm", "then", []string{}, "description")
```

Comment:
rethinking about that, I'm don't understand the name `then` of the metric
what's the usage of that metric?

Reply:
I believe this was just the previous name of the int value I was migrating over, but happy to use `last_check` since it seems to make sense. also for this value do we need to report it as a prometheus metric at all? my guess is this is just used for the agent status check?